packages/core/src/compiler/Style.ts

```diff
@@ -1790,15 +1791,32 @@ const resolveLhsName = (
   }
 };
 
+// const reservedVariableNames = im.Set(["match_id", "match_total"]);
+
 const resolveLhsPath = (
   block: BlockInfo,
   assignment: BlockAssignment,
   path: Path<C>
 ): Result<ResolvedPath<C>, StyleError> => {
   const { start, end, name, members, indices } = path;
-  return indices.length > 0
-    ? err({ tag: "AssignAccessError", path })
-    : ok({ start, end, ...resolveLhsName(block, assignment, name), members });
+  if (indices.length > 0) {
+    return err({ tag: "AssignAccessError", path });
+  } else {
+    const resolvedLhsName = resolveLhsName(block, assignment, name);
+    /*if (reservedVariableNames.includes(toString(name))) {
```

Comment:
Is this useful?

Reply:
If not, you can revert to the ternary right?

Reply:
Yup I can do that